domain-models-runtime/src/main/java/org/folio/rest/tools/utils/NetworkUtils.java

```diff
@@ -16,6 +16,7 @@
  */
 public class NetworkUtils {
 
+  @SuppressWarnings({"java:S2245","java:S2245"})//ThreadLocalRandom are not cryptographically secure, but performance is better in case of concurrent access.
```

Comment:
`ThreadLocalRandom are not cryptographically secure, but performance is better in case of concurrent access.`

I would replace it by

`Suppress "Weak Cryptography" warning, ThreadLocalRandom is good enough for finding a free port`